### DISCUSSION TITLE
Can I ignore specific words?

### DISCUSSION THREAD

**Discussion by fraserbooth** [WinMerge/winmerge#726] [Q&A] [answered]
I have been a keen WinMerge user for many years :)

Recently I've come across the need to be able to compare a series of folders where all the files have a couple of lines of code which are different, for thinks such as namespace, but the rest of the files are "mostly" the same. I would like to be able to tell WinMerge to ignore those unique words as being different, but to tell me if "anything else" has changed between them.

I've not come across this feature yet. Does anyone know how I might manage to achieve that? Maybe an additional Plugin somewhere?

Thank you for your advice :)

Fraser

**Comment by sdottaka** [comment, marked as answer]
Please use Substitution filters or PrediffLineFIlter.sct plugin

![image](https://user-images.githubusercontent.com/98126/114325692-651b6700-9b6c-11eb-9387-723ef530894b.png)

**Comment by fraserbooth** [comment]
Thank you, that works really well!
